examples/api/open-next.config.ts

```diff
@@ -0,0 +1,25 @@
+import type { OpenNextConfig } from "open-next/types/open-next";
+
+const config: OpenNextConfig = {
```

Comment:
It's probably a good idea to create some kind of function that could create the config object automatically and restrict what options could be set. 
I started implemented something like that here https://github.com/opennextjs/opennextjs-aws/blob/main/packages/open-next/src/helpers/withCloudflare.ts
This one won't work but something like that really specific to cloudflare

Reply:
Definitely something to add to the list. 
For now ensureClouflareConfig checks the required bits (the name of from memory)